testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -1724,7 +1724,19 @@ def test_set_dimensions(self):
         u.dimensions = np.array([10, 11, 12, 90, 90, 90])
         assert_allclose(u.dimensions, box)
 
+    def test_universe_kwargs(self):
+        u = MDAnalysis.Universe(PSF, PDB_small, fake_kwarg=True)
+        assert_equal(len(u.atoms), 3341, "Loading universe failed somehow")
+
+        assert_(u._kwargs['fake_kwarg'] is True)
```

Comment:
Adding a read-only property ``Universe.kwargs`` so we don't have to do private access generally. Should make this moot.